src/lib.rs

```diff
@@ -40,16 +40,27 @@ pub fn parse(contents: &str) -> Result<EditorConfigINIAST, Error<Rule>> {
 							ast.body.push(Item::Section(current_section.unwrap()));
 						}
 						current_section = Some(Section {
-							name: line.into_inner().next().unwrap().as_str().to_string(),
+							header: SectionHeader {
+								name: line.into_inner().next().unwrap().as_str().to_string(),
+								raws: Raws {
+									before: Some("".to_string()),
+									after: Some("".to_string()),
+									newline: Some("\n".to_string()),
+								},
+							},
 							body: Vec::new(),
 						});
 					}
-					Rule::property => {
+					Rule::pair => {
 						let mut inner_rules = line.into_inner();
-						let prop = Item::Property(Property {
+						let prop = Item::Pair(Pair {
 							name: inner_rules.next().unwrap().as_str().to_string(),
 							value: inner_rules.next().unwrap().as_str().to_string(),
-							newline: "\n".to_string(),
+							raws: Raws {
```

Comment:
This will make it easier to remove insignificants if they are all consistently stored as raws (stole this paradigm from PostCSS).

Reply:
Interesting - i have to take a look how they use it, too...

Reply:
https://github.com/postcss/postcss/blob/e0def602f546a86117cf7094831663dab7775341/lib/node.es6#L389-L390